helm/hpcc/templates/eclagent.yaml

```diff
@@ -50,20 +50,22 @@ data:
 {{ include "hpcc.addConfigMapVolumeMount" .me | indent 12 }}
 {{ include "hpcc.addDataVolumeMount" . | indent 12 }}
 {{ include "hpcc.addDllVolumeMount" .root | indent 12 }}
-{{ include "hpcc.addSecretVolumeMounts" (dict "root" .root "categories" (list "system" "ecl" "storage" ) ) | indent 12 }}
+{{ include "hpcc.addSecretVolumeMounts" (dict "root" .root "categories" .secretsCategoriesSystem ) | indent 12 }}
```

Comment:
I'm still not entirely clear on the distinction logical between vault secrets and non vault secrets.
(and their delination here into secretsCategoriesSystem(non-vault?) and secretsCategoriesUser(vault?)

I understand that vault as it stands has a different structure, and it will enable secrets to be read by individual ecl users (hence "ecl-user"), however any component that needs a "system" or "ecl" or "storage" secret could be under either implementation..

Reply:
@gavin should there be a single secretsCategories with ecl-user, ecl, storage and ecl-user?  Or should I be adding ecl-user to secretsCategoriesSystem?

Reply:
I think you should have a single set of categories

"system" "ecl-user" "ecl" "storage"

You could possibly use a category set of "system" for the qwait component if child processes are not being used, but I doubt it is worth distinguishing.

"system" is required by all components (see description in values.yaml)